infra/bridge/src/controllers/identity.js

```diff
@@ -84,26 +85,19 @@ router.post('/', identityWriteVerify, async (req, res) => {
   const data = req.body || {}
   logger.debug(`identity/write called for addr ${ethAddress}`)
 
-  if (!data.identity) {
+  const ipfsData = data.ipfsData
+  const ipfsHash = data.ipfsHash
+
+  if (!ipfsData) {
     return res.status(400).send({ errors: ['Identity data missing'] })
   }
-  if (!data.ipfsHash) {
+  if (!ipfsHash) {
     return res.status(400).send({ errors: ['IPFS hash missing'] })
   }
 
   // Parse the identity data to make sure it is valid.
   try {
-    validator(
-      'https://schema.originprotocol.com/identity_1.0.0.json',
-      data.identity
-    )
-    validator(
-      'https://schema.originprotocol.com/profile_2.0.0.json',
-      data.identity.profile
-    )
-    data.identity.attestations.forEach(a => {
-      validator('https://schema.originprotocol.com/attestation_1.0.0.json', a)
-    })
+    validateIdentityIpfsData(ipfsData)
```

Comment:
Out of curiosity, why the change from @origin/validator to this?  Shouldn't the JSON schema be the same?

Reply:
Just a refactor. validateIdentityIpfsData still uses @origin/validator to validate the data.